Lesson Plans/Instrument/IR.4 Flight Deck Organization, Workload and Automation Management.md

```diff
@@ -15,7 +15,7 @@ None
 - [[PPL.1 Aircraft Preflight, Taxiing, and Postflight Procedures]]
 
 ### Schedule
-Ground 1 hour 
+Ground 30 minutes
```

Comment:
seems to short with AP in this. Also maybe wants sim time to practice AP? @edwardabraham10 thoughts?

Reply:
Extended time a bit